packages/now-build-utils/src/detectors/eleventy.ts

```diff
@@ -1,13 +1,17 @@
 import { DetectorParameters, DetectorResult } from '../types';
 
 export default async function detectEleventy({
-  fs: { hasDependency },
+  fs: { hasDependency, getDependencyVersion },
 }: DetectorParameters): Promise<DetectorResult> {
   const hasEleventy = await hasDependency('@11ty/eleventy');
   if (!hasEleventy) return false;
   return {
     buildCommand: 'npx @11ty/eleventy',
```

Comment:
Did you intentionally leave out `getPackageJsonBuildCommand()` here?

Reply:
yes, because it uses `npx` by default